modules/rosetta/src/main/scala/org/tessellation/rosetta/domain/Operation.scala

```diff
@@ -0,0 +1,51 @@
+package org.tessellation.rosetta.domain
+
+import org.tessellation.ext.derevo.magnoliaCustomizable.snakeCaseConfiguration
+import org.tessellation.rosetta.domain.amount.Amount
+import derevo.cats.{eqv, show}
+import derevo.circe.magnolia._
+import derevo.derive
+import enumeratum.values.{StringCirceEnum, StringEnum, StringEnumEntry}
+import eu.timepit.refined.types.numeric.NonNegLong
+import io.circe.refined._
+import io.estatico.newtype.macros.newtype
+
+object operation {
+  @derive(customizableDecoder, customizableEncoder)
+  case class Operation(
+    operationIdentifier: OperationIdentifier,
+    relatedOperations: Option[List[OperationIdentifier]],
+    `type`: OperationType,
+    status: Option[OperationStatus],
+    account: Option[AccountIdentifier],
+    amount: Option[Amount]
+  )
+
+  @derive(customizableDecoder, customizableEncoder)
+  case class OperationIdentifier(
+    index: OperationIndex
+  )
+
+  @derive(eqv, show)
+  sealed abstract class OperationStatus(val value: String) extends StringEnumEntry
+
+  object OperationStatus extends StringEnum[OperationStatus] with StringCirceEnum[OperationStatus] {
+    val values = findValues
+
+    case object Accepted extends OperationStatus(value = "accepted")
+    case object Pending extends OperationStatus(value = "pending")
+    case object Rejected extends OperationStatus(value = "rejected")
+    case object Unknown extends OperationStatus(value = "unknown")
```

Comment:
![image](https://user-images.githubusercontent.com/4959931/218938592-e91cd133-6609-460c-a664-6a44067269c5.png)

Rosetta uses camel case for both `OperationStatus` and `OperationType`.
if this is network dependant and we can use any convention we want, I'd keep it unified.
see `accepted` and `TRANSFER`

Reply:
Standardized on camelcase.